### PR TITLE
updates to pointCNN guide - due to new features and workflow changes.

### PR DESCRIPTION
**Pull request by vbhv14**
- [x] added tips for new features.
- [x] removed obsolete tips.
- [x] added 3D DL tools info.
- [x] added deprecation info.
- [x] polished env. setup steps.
- [x] improved coherence about the term 'classification' and general grammar improvements.
- [x] guide name has to be changed from `point-cloud-segmentation-using-pointcnn` to `point-cloud-classification-using-pointcnn` without breaking the old URL. [this will be addressed via a separate PR]

### CODE REVIEW

**Comment by priyankatuteja** [comment]
@divyanshj16 Please have a look at this.

File: guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb [hunk 1]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```
Comment (divyanshj16) [inline guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb:4]: <p><code>Class preservation</code>  capital C looks odd to me. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

**Review by divyanshj16 — APPROVED** [review]
LGTM.

File: guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb (hunk 1 above)
Comment (vbhv14) [reply guide/14-deep-learning/point_cloud_segmentation_using_pointcnn.ipynb:4]: done

**Pull request merged**